dart-lsp/.claude-plugin/plugin.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "dart-lsp",
```

Comment:
Pls make sure directory and name here is the same as in root marketplace.json -> dart-analyzer